lib/app/features/feed/views/pages/feed_page/components/feed_controls/components/feed_navigation/feed_notifications_button.dart

```diff
@@ -15,21 +16,23 @@ class FeedNotificationsButton extends ConsumerWidget {
 
   @override
   Widget build(BuildContext context, WidgetRef ref) {
-    return Stack(
-      clipBehavior: Clip.none,
-      children: [
-        NavigationButton(
-          onPressed: () => NotificationsHistoryRoute().push<void>(context),
-          icon: Assets.svg.iconHomeNotification.icon(
-            color: context.theme.appColors.primaryText,
+    return ListCachedObjectsWrapper(
```

Comment:
Can u please add a comment why do we need it here, cuz or the lists it's obvious but here it looks like a hack